web/releases/tenzir-node-v4.31/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Tenzir Node v4.31: OpenSearch Ingestion and if-expressions"
+slug: tenzir-node-v4.31
+authors: [IyeOnline, raxyte]
+date: 2025-03-31
+tags: [release, node]
+comments: true
+---
+
+[Tenzir Node v4.31][github-release] is now available, with several new features
+including OpenSearch emulation for broader data ingestion, flexible if/else
+expressions in TQL, and a new operator for writing Syslog messages.
+
+![Tenzir Node v4.31](tenzir-node-v4.31.svg)
+
+[github-release]: https://github.com/tenzir/tenzir/releases/tag/v4.31.0
+
+<!-- truncate -->
+
+## Emulating OpenSearch
+
+Tenzir can now receive data from a variety of tools such as
+[Beats](https://www.elastic.co/beats) using their OpenSearch/Elasticsearch
+outputs via the new [`from_opensearch`](/next/tql2/operators/from_opensearch)
+operator.
+
+For example, to receive events from [Filebeat](https://www.elastic.co/beats/filebeat):
+
+Setup an Elasticsearch sink for Filebeat:
+
+```yml title="filebeat.yml"
+output.elasticsearch:
+  hosts: ["<host to forward to>"]
+```
+
+and listen on the host with:
+
+```tql
+from_opensearch keep_actions=true
```

Comment:
How does this work without listening host/IP?

Reply:
It defaults to the localhost:9200, which is the default for Opensearch and ElasticSearch as well.